### PR TITLE
Fix #221 - markdown snippets check with mdoc

### LINKED ISSUE #221: Find a way to typecheck the snippets in markdown
Ideally we'd type check the snippets in readme's and on the website.

### PR DESCRIPTION
**Pull request by pawelprazak**
as in title

### CODE REVIEW

File: Justfile [hunk 1]
```diff
@@ -309,6 +309,20 @@ test-examples:
...
 	for file in `ls -d examples/*/ | cut -f2 -d'/'`; do just clean-test-example $file; done
 
+####################
+# Website and docs
+####################
+
+# Runs tests for website and docs
+test-markdown:
+	cs launch org.scalameta:mdoc_2.12:2.3.8 -- --in ./README.md ./CONTRIBUTING.md --out target/mdoc-readme --site.version=$(cat version.txt)
+	cs launch org.scalameta:mdoc_2.12:2.3.8 -- --in ./website --out target/mdoc-website --exclude node_modules --site.version=$(cat version.txt)
```
Comment (KacperFKorban) [inline Justfile:319]: Does it actually test the snippets? Don't we actually need to add the dependencies etc.?
Also shouldn't the snippets use the `mdoc` annotations? I'm not an mdoc expert, but the snippet in the guide has it.

Comment (pawelprazak) [reply Justfile:319]: not an expert on mdco either, but this works :)

Comment (KacperFKorban) [reply Justfile:319]: But did u manage to make it give a compilation error for any snippet?

Comment (pawelprazak) [reply Justfile:319]: yes, it gave an error :)

**Review by KacperFKorban — APPROVED** [review]

**Pull request merged**